deimos/openssl/ssl.d

```diff
@@ -1930,6 +1930,10 @@ const(SSL_METHOD)* SSLv3_method();		/* SSLv3 */
 const(SSL_METHOD)* SSLv3_server_method();	/* SSLv3 */
 const(SSL_METHOD)* SSLv3_client_method();	/* SSLv3 */
 
+const(SSL_METHOD)* SSLv23_method();        /* SSLv3 but can rollback to v2 */
+const(SSL_METHOD)* SSLv23_server_method(); /* SSLv3 but can rollback to v2 */
+const(SSL_METHOD)* SSLv23_client_method(); /* SSLv3 but can rollback to v2 */
+
```

Comment:
These do not match the checked-in version of the C headers.

There, these are defined as aliases of `TLS_*` functions:

```c
#define SSLv23_method           TLS_method
#define SSLv23_server_method    TLS_server_method
#define SSLv23_client_method    TLS_client_method
```

If these bindings are for a different version of OpenSSL, then they ought to go into a branch for that version.
If the C bindings in the repository are outdated and don't match the OpenSSL versions suggested by the tags, then they should be fixed as well.